src/plugins/dashboard/public/dashboard_app/_dashboard_app_strings.ts

```diff
@@ -26,9 +26,17 @@ export const dashboardReadonlyBadge = {
 };
 
 export const dashboardManagedBadge = {
-  getTooltip: () =>
+  getText: () =>
     i18n.translate('dashboard.badge.managed.tooltip', {
-      defaultMessage: 'Elastic manages this dashboard. Clone it to make changes.',
+      defaultMessage: 'Elastic manages this dashboard.',
+    }),
+  getDuplicateText: () =>
+    i18n.translate('dashboard.managedContentPopoverFooterText', {
+      defaultMessage: 'Duplicate it to make changes.',
+    }),
```

Comment:
Can we call this `getDuplicateButtonAriaLabel` to be more specific on what we are using it for? And perhaps the text should be changed to something like `"Click here to duplicate this dashboard."`